target_snowflake/__init__.py

```diff
@@ -167,7 +170,18 @@ def persist_lines(config, lines, table_cache=None, file_format_type: FileFormatT
             else:
                 records_to_load[stream][primary_key_string] = o['record']
 
+            flush = False
             if row_count[stream] >= batch_size_rows:
+                flush = True
+                LOGGER.info("Flushing %s rows, triggered by batch_size_rows (%s)",
```

Comment:
This log message is not always necessarily accurate, and I'm afraid it would cause confusion.
Not all rows are gonna be flushed, ie `sum(row_count.values())`, by default only the stream (table) that reach `batch_size_rows` will be flushed only and rest would still be in memory. Flushing rows of all streams would only happen if `flush_all_streams: true` in the config.

Reply:
@Samira-El thanks, revised the wording